javaslang/src-gen/test/java/javaslang/APITest.java

```diff
@@ -165,6 +165,16 @@ public void shouldTuple8ReturnNotNull() {
         assertThat(tuple(1, 2, 3, 4, 5, 6, 7, 8)).isNotNull();
     }
 
+    @Test
+    public void shouldRightReturnNotNull() {
+        assertThat(right(null)).isNotNull();
```

Comment:
API, as described before, can be used as single static import `import static javaslang.API.*` - and you receive access to almost functions from Javaslang, without needed to import something else